_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -93841,7 +93857,8 @@
 	start_active = 1
 	},
 /turf/open/floor/bluegrid{
```

Comment:
These floors shouldn't be AI core floors, it makes no sense. Something like the kitchen's meat room floors would make more sense.